lib/reek/cli/silencer.rb

```diff
@@ -4,10 +4,13 @@ module CLI
     # @api private
     module Silencer
       def self.silently
-        old_verbose, $VERBOSE = $VERBOSE, nil
-        yield if block_given?
+        original_stderr = $stderr
+        original_stdout = $stdout
+        $stderr = $stdout = StringIO.new
```

Comment:
Hmmm. It seems you need this version (that gulps stdout/stderr, rather than turning off warnings) to use in specs – but wouldn’t using this in specs mean that any spec failures won’t show on spec runs?